modules/theme/src/themes/default/views/card.variables

```diff
@@ -33,5 +33,5 @@
 @settingsCardActionButtonColor: @primaryColor;
 @settingsCardTabItemBorderBottom: 3px;
 @settingsCardTabItemBorderBottomColor: @primaryColor;
-@settingsCardTopActionPanelBorderTop: @gallery;
-@settingsCardTopActionPanelBorderBottom: @gallery;
+@settingsCardTopActionPanelBorderTop: 1px solid @gallery;
```

Comment:
Let's makes the color `gallery-grey` instead

Reply:
This variable was renamed to @galleryGray